docs/en/rules/Azure.Redis.Version.md

```diff
@@ -0,0 +1,88 @@
+---
+severity: Important
+pillar: Reliability
+category: Requirements
+resource: Azure Cache for Redis
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Redis.Version/
+---
+
+# Redis version for Azure Cache for Redis
+
+## SYNOPSIS
+
+Azure Cache for Redis should use the latest supported version of Redis.
+
+## DESCRIPTION
+
+Azure Cache for Redis supports Redis 6. Redis 6 brings new security features and better performance.
+
+Version 4 for Azure Cache for Redis instances will be retired on June 30, 3023.
+
+## RECOMMENDATION
+
+Consider upgrading Redis version for Azure Cache for Redis to the latest supported version (>=6.0).
+
+## EXAMPLES
+
+### Configure with Azure template
+
+To deploy Azure Cache for Redis instances that pass this rule:
+
+- Set the `properties.redisVersion` property to `latest` or `>=6`.
```

Comment:
I think this may confuse some customers because the template/ Bicep code won't accept this literally. Try:

```suggestion
- Set the `properties.redisVersion` property to `latest` or `6`.
```